schema/verify/policies/naics_policies.sql

```diff
@@ -0,0 +1,7 @@
+-- Verify ggircs-portal:policies/naics_policies on pg
+
+BEGIN;
+
+-- XXX Add verifications here.
+
```

Comment:
any verify needed here?

Reply:
policy verify files (and really, all non-schema related verifies) are...odd & should maybe just be in the scope of the unit tests. @pbastia and I ran into an issue with verifying policies where we either need to update all the verify files (including previous migrations where we used the `_001` method & not a `rework`) when we remove policies or agree that the policies are out of scope of the verify & should be the job of the tests (All tables have RLS tests that will fail if a necessary policy is missing). Sequentially verifying the files during a deploy works fine, but the `sqitch verify` command only works over the final state of the db and will kerplode if a non-reworked verify file asserts that a policy should exist and it doesn't in the final state.

example:
**policy_a_verify**
- assert x exists
- asset y exists
**policy_a_001_verify** (deploy file removes x)
- assert y exists

Because `policy_a` is different than `policy_a_001` in sqitch's mind, it runs both verify files & when it sees x does not exist, throws an exception in the `policy_a_verify` file. We may want to rethink what goes in a verify file (schema related only?) and what should be checked in tests, especially when using the `_001` to perform migrations.

Reply:
This particular verify file is going away though, since we don't need policies for this table :P